src/System.Security.Cryptography.Encoding/src/Internal/Cryptography/OpenSslAsnFormatter.cs

```diff
@@ -48,11 +48,11 @@ protected override string FormatNative(Oid oid, byte[] rawData, bool multiLine)
                         return null;
                     }
 
-                    int printLen = Interop.libcrypto.GetMemoryBioSize(bio);
+                    int printLen = Interop.Crypto.GetMemoryBioSize(bio);
 
                     // Account for the null terminator that it'll want to write.
                     StringBuilder builder = new StringBuilder(printLen + 1);
-                    Interop.libcrypto.BIO_gets(bio, builder, builder.Capacity);
+                    Interop.Crypto.BioGets(bio, builder, builder.Capacity);
```

Comment:
@bartonjs, separate from this change, we should audit the various places StringBuilder is being used in these signatures.  All of the uses of BioGets, for example, are of the form:
```C#
StringBuilder sb = new StringBuilder(size);
NativeCall(sb, sb.Capacity);
return sb.ToString();
```
This not only allocates the underlying char[], but also the StringBuilder object, as well as additional copy costs to marshal out and in the current StringBuilder implementation another char[] allocation on the way back.  Seems it could be implemented just as easily as:
```C#
char[] chars = new char[size];
NativeCall(chars, chars.Length);
return Encoding.UTF8.GetString(chars);
```
or something like that.